playbooks/sync-task.yml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
@Metfriet, you will have to create a new role, add the below tasks in `tasks/main.yml` and add the template file to the `templates` directory. You can also define the variables and the values in `defaults/main.yml` Then you will have to simplify this playbook to just include that role.

Reply:
@lgp171188 I've created a role folder and relevant files. Is it ok like this do you think?